immune_subtype_classifier/prepare_data_sheet.py

```diff
@@ -92,11 +106,14 @@ def syn_upload(
 
 def main():
     parent = sys.argv[1]  # "syn26535390"
-    export_name = sys.argv[2]  # "immune_subtype_sample_sheet.tsv"
-    upload_location = sys.argv[3]  # "syn51471781"
+    map_file_id = sys.argv[2]  # "syn51526489"
+    export_name = sys.argv[3]  # "immune_subtype_sample_sheet.tsv"
+    upload_location = sys.argv[4]  # "syn51526284"
     syn = syn_login()
     file_entity_list = download_data_files(parent=parent, syn=syn)
-    gene_df_list = load_data_files(file_entity_list=file_entity_list)
+    gene_df_list = load_data_files(
+        file_entity_list=file_entity_list, map_file_id=map_file_id, syn=syn
```

Comment:
Can you add the map_file_id to the provenance so we can track it?

Reply:
added in latest commit